spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -167,8 +179,11 @@ class Graph[T: ClassTag](val inputs : Seq[ModuleNode[T]],
       })
 
       gradOutputBP(i) = curGradOutput
-      curNode.element.updateGradInput(inputsBP(i), curGradOutput)
-      i -= 1
+
+      if (!curNode.element.isStopGradient()) {
```

Comment:
why we need this? I think we already filter this layer in the backwardExecution generation